sessionctx/variable/session.go

```diff
@@ -289,6 +289,9 @@ type SessionVars struct {
 	// EnableTablePartition enables table partition feature.
 	EnableTablePartition bool
 
+	// EnableCascadesPlanner enables the cascades planner.
+	EnableCascadesPlanner bool
```

Comment:
do we need half-open stauts that plan/exec new & old a sametime for compare and test?